modules/using-assisted-installer-oci-agent-iso.adoc

```diff
@@ -0,0 +1,47 @@
+* If you use a firewall and you plan to use a Telemetry service, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured your firewall], so that {product-title} can access the sites required.
+* Before you create a virtual machines (VM), refer to {op-system-base-full} certified shapes, instance types, to identify the supported {oci} VM shapes. See link:https://catalog.redhat.com/cloud/detail/216977[Cloud instance types] on Red Hat Ecosystem Catalog portal.
+
+.Procedure
+
+. From the link:https://console.redhat.com/openshift/assisted-installer/clusters/~new[**Install OpenShift with the Assisted Installer**] page on the Hybrid Cloud Console, generate the discovery ISO image by completing all the required Assisted Installer steps. 
++
+.. For the **Cluster Details** step, you complete the following fields:
++
+... **Cluster name**: Specify the name of your cluster, such as `ocidemo`. 
+... **Base domain**: Specify the base domain of the cluster, such as `splat-oci.devcluster.openshift.com`. Provided you previously created a compartment on {oci}, you can get this information by going to **DNS management > Zones > List scope** and then select the parent compartment. Your base domain should show under the **Public zones** tab.
+... **OpenShift version**: Specify OpenShift 4.15 or a later version.
+... **CPU architecture**: Specify `x86_64` or `Arm64`.
+... **Integrate with external partner platforms**, where you must specify `Oracle Cloud Infrastructure`.
++
+After you specify this value, the *Include custom manifests* checkbox is automatically selected. 
++
+[IMPORTANT] 
+====
+You can keep the default settings for the **Operators** step.
+====
+
+. For the **Host Discovery** step, click on the **Add hosts** button to display a dialog box. For the **SSH public key** field, add your SSH key from your local system. Click the **Generate Discovery ISO** button to generate the discovery image ISO file. Ensure that you download the file to your local system.
++
+[TIP]
+====
+You can create an SSH authentication key pair by using the `ssh-keygen` tool. 
+====
+
+. On {oci}, go to **Resources > Objects** and then click the **Upload** button. Upload the Agent ISO image to the bucket, and then click the **Upload** button to complete the upload operation.
+
+. From the **Objects** menu, locate your uploaded image, and then click the overflow menu. From the **Create Pre-Authenticated Request** window, select the **Object** tile. After you create the request, copy the URL from the **Pre-Authenticated Request URL** field. See link:https://docs.oracle.com/en-us/iaas/Content/Object/Tasks/usingpreauthenticatedrequests_topic-To_create_a_preauthenticated_request_for_all_objects_in_a_bucket.htm#:~:text=Open%20the%20navigation%20menu%20and,Click%20Create%20Pre%2DAuthenticated%20Request.[Creating a Pre-Authenticated Request] in the Oracle documentation. 
```

Comment:
Unfortunately this link redirect to empty page. Need update with correct one. 
Probably this one
https://docs.oracle.com/en-us/iaas/Content/Object/Tasks/usingpreauthenticatedrequests_topic-To_create_a_preauthenticated_request_for_all_objects_in_a_bucket.htm